src/dotnet-svcutil/lib/src/CodeSerializer.cs

```diff
@@ -0,0 +1,191 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
```

Comment:
The copyright is wrong on all these files. It should be dotnet foundation that holds the copyright, not Microsoft.